Contributors.md

```diff
@@ -9,14 +9,14 @@ The following people have contributed to the SCAP Security Guide project
 * Frank J Cameron (CAM1244) <cameron@ctc.com>
 * 0x66656c6978 <0x66656c6978@users.noreply.github.com>
 * Gabe Alford <redhatrises@gmail.com>
+* Firas AlShafei <firas.alshafei@us.abb.com>
 * Christopher Anderson <cba@fedoraproject.org>
 * Chuck Atkins <chuck.atkins@kitware.com>
 * Ryan Ballanger <root@rballang-admin-2.fastenal.com>
 * Molly Jo Bault <Molly.Jo.Bault@ballardtech.com>
 * Jose Luis BG <bgjoseluis@gmail.com>
 * Joseph Bisch <joseph.bisch@gmail.com>
 * Jeffrey Blank <blank@eclipse.ncsc.mil>
-* Olivier Bonhomme <ptitoliv@ptitoliv.net>
```

Comment:
?

Reply:
He is added later, but just as Olivier.
His name might have changed in later commits.

Reply:
Full name is better than just first name, what about creating an alias.

Reply:
Good idea, name mapped.